distribution/debian/install.sh

```diff
@@ -159,23 +260,48 @@ Restart=on-failure
 [Install]
 WantedBy=multi-user.target
 EOF
+sleep 2
+echo ""
+echo -e "New service file created!"
 
 # Start the App
-echo "Service file created. Attempting to start the app"
+echo ""
+echo -e "${brown}[${app^}]${reset} is attempting to start, this may take a few seconds..."
 systemctl -q daemon-reload
 systemctl enable --now -q "$app"
+sleep 3
+
+# Check if the service is up and running
+echo ""
+echo "Checking if the service is up and running..."
+# Loop to wait until the service is active
+while ! systemctl is-active --quiet "$app"; do
+    sleep 1
+done
```

Comment:
What if for some reason the app isn't able to start? The loop will get stuck here forever?
perhaps introducing a counter with a max waiting time would be good?